tools/BugReportTool/BugReportTool/RegistryUtils.cpp

```diff
@@ -282,7 +282,7 @@ void ReportRegistry(const filesystem::path& tmpDir)
                 {
                     DWORD data = 0;
                     DWORD dataSize = sizeof(data);
-                    LONG retCode = RegGetValueW(rootKey, subKey.c_str(), value.c_str(), flags, &type, &data, &dataSize);
+                    RegGetValueW(rootKey, subKey.c_str(), value.c_str(), flags, &type, &data, &dataSize);
```

Comment:
Should be `result` used here instead of `retcode`. Likely it was a copy paste mistake.

Reply:
I think you are right! I will change it to result